content/en/real_user_monitoring/guide/enable-rum-woocommerce-store.md

```diff
@@ -49,15 +49,13 @@ For example, you can:
 
 - Get valuable insights about your customers' behavior by
 making data-driven decisions to improve your store
-- Increase conversion by watching browser recordings enriched sessions with [Session Replay][2]
-- Use [funnel analysis][3] to better understand the customer journey, or
-- [Generate metrics][4] from those newly captured sessions
+- Increase conversion by watching browser recordings enriched sessions with [Session Replay][2] or
```

Comment:
```suggestion
- Increase conversion by watching browser recordings of enriched sessions with [Session Replay][2] or
```
I know this was already here, but it seems like there's a word missing? I'll approve it in case I'm wrong and you still want to merge it as-is.

Reply:
Great catch @jhgilbert, thanks!